k8s/images/nginx/download_sigil.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+export SIGIL_VERSION=0.10.1
+export OS=`sh -c "uname -s | tr '[:upper:]' '[:lower:]'"`
+export ARCH=`sh -c "uname -m | tr '[:upper:]' '[:lower:]' | sed 's/aarch64/arm64/'"`
+
+
+curl -L "https://github.com/gliderlabs/sigil/releases/download/v${SIGIL_VERSION}/gliderlabs-sigil_${SIGIL_VERSION}_${OS}_${ARCH}.tgz" | tar -zxC /tmp
+mv /tmp/gliderlabs-sigil-${ARCH} /tmp/sigil
```

Comment:
I see a refactor in our future lol

Reply:
I actually didn't carry out the refactor we talked about, where we can simply dispense with Sigil. Should I do that in this PR @DXCanas ?

Reply:
Nah. Too big a load, and this should be fixed ASAP. We should also think about the various paths forward. Hence my asking @bjester  for his thoughts on moving some of this to infra:
https://github.com/learningequality/studio/pull/4336#issuecomment-1808813010